index.html

```diff
@@ -4,8 +4,32 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Classic Resume</title>
+    <link
+    rel="stylesheet"
+    href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.1/css/all.min.css"
```

Comment:
When you import a whole css library made of fonts or icons that you don't need, it hits on the performance of your site. A better way would be to copy the SVG of the icon you need from an online library or online icon sites.